exporter/stats/stackdriver/stackdriver_test.go

```diff
@@ -249,6 +250,39 @@ func TestEqualAggWindowTagKeys(t *testing.T) {
 	}
 }
 
+func TestSanitize(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    string
+		expected string
+	}{
+		{
+			name:     "trunacate long string",
+			input:    strings.Repeat("a", 101),
+			expected: strings.Repeat("a", 100),
+		},
+		{
+			name:     "replace character",
+			input:    "test/key-1",
+			expected: "test_key_1",
+		},
+		{
+			name:     "don't modify alphanumeric string",
+			input:    "testkey1",
+			expected: "testkey1",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			actual := sanitize(tt.input)
```

Comment:
if got, want := sanitize(tt.input), tt.want; got != want {
      t.Errorf("sanitize() = %q; want %q", got, want)
}

got/want is the jargon in Go and use %q to escape.